### PR TITLE
Driver status of CMAC: accelerate CMAC in accel_cipher

### PR DESCRIPTION
**Pull request by yanrayw**
## Description

Resolve: #8314 

## PR checklist

- [x] **changelog** not required
- [x] **backport** not required - this is mostly about improving the driver test framework to support new development.
- [x] **tests** not required

### CODE REVIEW

**Comment by mpg** [comment]
Removing from the board as it's already linked to the issue that's on the board - we don't need the PR to appear twice there.

**Review by mpg — APPROVED** [review]
LGTM, thanks!

**Review by valeriosetti — APPROVED** [review]
LGTM

File: tests/scripts/all.sh [hunk 1]
```diff
@@ -3528,22 +3528,22 @@ component_test_psa_crypto_config_reference_hash_use_psa() {
...
 
-    loc_accel_list="ALG_CBC_NO_PADDING ALG_CBC_PKCS7 ALG_CTR ALG_CFB ALG_OFB ALG_XTS KEY_TYPE_DES"
+    loc_accel_list="ALG_CBC_NO_PADDING ALG_CBC_PKCS7 \
+                    ALG_CTR ALG_CFB ALG_OFB ALG_XTS \
+                    KEY_TYPE_DES ALG_CMAC"
 
     # Configure
     # ---------
 
+    # There is no intended accelerator support for STREAM_CIPHER and
```
[the remark this answers is not in the document]
Comment (mpg) [reply tests/scripts/all.sh:3538]: I've pressed the merge button on this, so I guess 8368 will need rebasing when this one's merged.

**Pull request merged**